src/pages/admin/AdminCustomerInstitutionPage.tsx

```diff
@@ -1,10 +1,161 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import Card from '../../components/Card';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import { Field, Form, Formik } from 'formik';
+import * as Yup from 'yup';
+import {
+  CustomerInstitutionFieldNames,
+  emptyCustomerInstitution,
+  emptyInstitutionLogoFile,
+  InstitutionLogoFile,
+} from '../../types/customerInstitution.types';
+import { Button } from '@material-ui/core';
+import { TextField } from 'formik-material-ui';
+import Heading from '../../components/Heading';
+import UppyDashboard from '../../components/UppyDashboard';
+import { createUppy } from '../../utils/uppy-config';
+import Label from '../../components/Label';
+
+const StyledField = styled(Field)`
+  margin: 1rem;
+  flex: 1 0 40%;
+`;
+
+const StyledButtonContainer = styled.div`
+  margin-top: 2rem;
+  margin-right: 1rem;
+  display: flex;
+  justify-content: flex-end;
+`;
 
 const AdminCustomerInstitutionPage: FC = () => {
+  const { t } = useTranslation('admin');
+  const shouldAllowMultipleFiles = false;
```

Comment:
flytt denne utenfor komponenten, eller bare send inn false som argument direkte, så slipper vi å opprette variabelen på nytt for hver render?